float64 tachoVelocity
float64 tachoVelocityOld
